tests/Menu.spec.tsx

```diff
@@ -730,5 +730,20 @@ describe('Menu', () => {
 
     expect(document.activeElement).toBe(last(container.querySelectorAll('li')));
   });
+  it('should render a divider with role="separator"', () => {
+    const menuRef = React.createRef<MenuRef>();
+    const { container } = render(
+      <Menu ref={menuRef} activeKey="cat">
+        <MenuItem key="light">Light</MenuItem>
+        <Divider />
+        <MenuItem key="cat">Cat</MenuItem>
+      </Menu>,
+    );
+    // Get the separator element
+    const separator = container.querySelector('[role="separator"]');
```

Comment:
It would be better to query all `rc-menu-item-dividers` to check that each one contains the correct role.